Tori.teng/pattern exclusion filter (#13475)
DataDog/documentation: content/en/logs/guide/getting-started-lwl.md

**To create an exclusion filter**:

1. Click on a pattern from the pattern view list.

> 2. Click the **Add Exclusion Filter** button in the top right corner. This button will be disabled if less than half of the logs in this pattern fall into a single index.

Reviewer: ```suggestion
2. Click the **Add Exclusion Filter** button in the top right corner. This button is disabled if less than half of the logs in this pattern fall into a single index.
```

> 3. Upon clicking the button, the Log Index Configuration page will be opened in a new tab with a pre-filled exclusion filter for the index where majority of the logs for that pattern reside.

Reviewer: ```suggestion
3. The Log Index Configuration page opens in a new tab with a pre-filled exclusion filter for the index where the majority of the logs for that pattern show up.
```

4. The exclusion filter will be populated with an automatically generated search query associated with this pattern. Input the filter name and set an exclusion percentage before saving the new exclusion filter.

{{< img src="logs/guide/getting-started-lwl/exclusion_filter.mp4" alt="Exclusion Filter" video=true style="width:100%;">}}

**Note**: If a log matches several exclusion filters, only the first exclusion filter rule is applied. A log is not sampled or excluded multiple times by different exclusion filters.

In this example, the service `email-api-py`, status `INFO` pattern `response code from ses 200` is filtered with an exclusion filter. Removing any high volume logging pattern similar to this one from Log Explorer helps you reduce noise and identify issues quicker. However, these logs are **only** excluded from indexing. They are still ingested and available to view in [Live Tail][5] and can be sent to [log archives][6] or used to [generate metrics][7].

> {{< img src="logs/guide/getting-started-lwl/live_tail.png" alt="Live Tail" style="width:100%;">}}

Reviewer: ```suggestion
{{< img src="logs/guide/getting-started-lwl/live_tail.png" alt="The Live Tail page showing a list of logs and the time frame dropdown" style="width:100%;">}}
```
